DSCResources/MSFT_xDSCWebService/SecureTLSProtocols.psm1

```diff
@@ -0,0 +1,48 @@
+# This module file contains a utility to test and set TLS protocol according best practices
+#
+# Copyright (c) Microsoft Corporation, 2016
+#
+
+# ============ Best Practice Security Settings Block =========
+$insecureProtocols            = @("SSL 2.0", "SSL 3.0", "TLS 1.0", "PCT 1.0", "Multi-Protocol Unified Hello")
+$secureProtocols              = @("TLS 1.1", "TLS 1.2")
+
+# ===========================================================
+
+function Test-Protocol
```

Comment:
Thanks!

---
In reply to: [85399974](https://github.com/PowerShell/xPSDesiredStateConfiguration/pull/261#discussion_r85399974) [](ancestors = 85399974)